.gitignore

```diff
@@ -2,7 +2,8 @@
 *.log
 
 # sbt specific
-build.properties
+metals*
```

Comment:
should be `.metals` I think

Reply:
Yes, small oversight: ad740ef0e33973907961077f7868de12b2383623